src/consts.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
JSdoc don't know how to display var in a file?

Reply:
I didn't find another way to do that.

Reply:
https://stackoverflow.com/questions/19093935/how-to-document-a-string-type-in-jsdoc-with-limited-possible-values maybe?

Reply:
OK, I checked only how it generates README.md file, but actually it's more valuable in IDEs, and `enum` is not perfect in generated doc, but in WebStorm it looks much better:

param:
![image](https://user-images.githubusercontent.com/5202105/62186884-ef577380-b367-11e9-8766-db2fde141fbf.png)

allowed values:
![image](https://user-images.githubusercontent.com/5202105/62186908-04cc9d80-b368-11e9-8345-5484f62b930a.png)